source/assets/js/playground/autocomplete.ts

```diff
@@ -6,7 +6,15 @@ import {
 import {sassCompletionSource} from '@codemirror/lang-sass';
 import {syntaxTree} from '@codemirror/language';
 import {EditorState} from '@codemirror/state';
-import moduleMembers from './module-members';
+import type {ModuleDefinition} from './generate-module-members';
+
+let moduleMembers: ModuleDefinition[] = [];
+try {
+  moduleMembers = require('./module-members');
+} catch (error) {
+  console.error('module-members.ts is missing');
+  throw error;
+}
```

Comment:
@jamesnw Do you think this is worth it? It basically just allows the linter to run fine without needing the generated file. 🤷